cli/src/lib.rs

```diff
@@ -245,6 +295,7 @@ impl Iroha {
             kura,
             block_sync,
             torii,
+            sumeragi_relay,
         })
     }
 
```

Comment:
Failed tests as of 0390bd29 compared to 03ef82b1:

- [ ] `integration::asset_propagation::client_add_asset_quantity_to_existing_asset_should_increase_asset_amount_on_another_peer`
- [ ] `integration::connected_peers::connected_peers_with_f_1_0_1`
- [ ] `integration::connected_peers::connected_peers_with_f_2_1_2`
- [ ] `integration::events::pipeline::transaction_with_fail_instruction_should_be_rejected`
- [ ] `integration::events::pipeline::transaction_with_no_instructions_should_be_committed`
- [ ] `integration::multisignature_transaction::multisignature_transactions_should_wait_for_all_signatures`
- [ ] `integration::unregister_peer::unstable_network_stable_after_add_and_after_remove_peer`

If it's intentional, I'd like an explanation

Reply:
In order to allow multiple people to work on the reimplementation, this PR has things that are broken. Then we will fix these with more PR's into iroha2-staging. We only merge into iroha2-dev when the reimplementation is complete and everything works. Aleksandr for instance is working on #2561 which fixes most of the tests you mention. And if some are still failing after that we will fix them.